src/app/shared/components/navbar-mobile/navbar-mobile-dialog/navbar-mobile-dialog.component.html

```diff
@@ -1,71 +1,89 @@
     </button>
   </mat-card-header>
   <mat-card-content class="navbar-mobile__dialog__content">
-    <hr />
-    <a
-      [queryParams]="{}"
-      [routerLinkActiveOptions]="{exact: true}"
-      class="navbar__item"
-      mat-button
-      [routerLink]="[mainPageEnum.Start]"
-      routerLinkActive="navbar__item--active"
-      ariaCurrentWhenActive="page"
-      (click)="close()"
-      >Geoportal</a
-    >
-    <hr />
-    <a
-      class="navbar__item"
-      mat-button
-      [routerLink]="[mainPageEnum.Maps]"
-      routerLinkActive="navbar__item--active"
-      ariaCurrentWhenActive="page"
-      (click)="close()"
-      >GIS-Browser</a
-    >
-    <hr />
-    <a
-      class="navbar__item"
-      mat-button
-      [routerLink]="[mainPageEnum.Data]"
-      routerLinkActive="navbar__item--active"
-      ariaCurrentWhenActive="page"
-      (click)="close()"
-      >Geodatenkatalog</a
-    >
-    <hr />
-    <a
-      class="navbar__item"
-      mat-button
-      [routerLink]="[mainPageEnum.Support]"
-      routerLinkActive="navbar__item--active"
-      ariaCurrentWhenActive="page"
-      (click)="close()"
-      >Hilfe & Support</a
-    >
-    <hr />
-    <a class="navbar__item" href="https://geodatenshop.zh.ch" mat-button target="_blank" rel="noopener noreferrer" (click)="close()">
-      <mat-icon>launch</mat-icon>
-      Geodatenshop</a
-    >
-    <hr />
+    <div class="navbar-mobile__dialog__content__container">
+      <a
+        [queryParams]="{}"
+        [routerLinkActiveOptions]="{exact: true}"
+        class="navbar__item"
+        mat-button
+        [routerLink]="[mainPageEnum.Start]"
+        routerLinkActive="navbar__item--active"
+        ariaCurrentWhenActive="page"
+        (click)="close()"
+        >Geoportal</a
+      >
+      <a
+        class="navbar__item"
+        mat-button
+        [routerLink]="[mainPageEnum.Maps]"
+        routerLinkActive="navbar__item--active"
+        ariaCurrentWhenActive="page"
+        (click)="close()"
+        >GIS-Browser</a
+      >
+      <a
+        class="navbar__item"
+        mat-button
+        [routerLink]="[mainPageEnum.Data]"
+        routerLinkActive="navbar__item--active"
+        ariaCurrentWhenActive="page"
+        (click)="close()"
+        >Geodatenkatalog</a
+      >
+      <a
+        class="navbar__item"
+        mat-button
+        [routerLink]="[mainPageEnum.Support]"
+        routerLinkActive="navbar__item--active"
+        ariaCurrentWhenActive="page"
+        (click)="close()"
+        >Hilfe & Support</a
+      >
+      <a class="navbar__item" href="https://geodatenshop.zh.ch" mat-button target="_blank" rel="noopener noreferrer" (click)="close()">
+        <mat-icon>launch</mat-icon>
+        Geodatenshop</a
+      >
+      <a
+        class="navbar__item"
+        href="https://gdi.zh.ch/portal/home/"
+        mat-button
+        target="_blank"
+        rel="noopener noreferrer"
+        matTooltip="Kollaborationsplattform"
```

Comment:
I'm not sure what we decided here: Shall we leave this in or don't use tooltips until they get requested?

Reply:
removed